corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -271,11 +294,22 @@ def _with_progress(self, doc_types, iterable, progress_name='Migrating'):
                 get_doc_count_in_domain_by_type(self.domain, doc_type, XFormInstance.get_db())
                 for doc_type in doc_types
             ])
+            setattr(self, self.timing_name + '_doc_count', doc_count)
             prefix = "{} ({})".format(progress_name, ', '.join(doc_types))
             return with_progress_bar(iterable, doc_count, prefix=prefix, oneline=False)
         else:
             return iterable
 
+    def _send_timings(self, timing_context):
+        metric_name_template = "commcare.%s"
+        metric_name_template_normalized = "commcare.%s.normalized"
+        for timing in timing_context.to_list():
+            datadog_histogram(metric_name_template % timing.full_name, timing.duration)
```

Comment:
Datadog histograms are aggregated on a fixed interval (10s), so I'm not sure this will do what you want. [Here's a resource](https://confluence.dimagi.com/x/4ArDAQ#TroubleshootingPasteboard/HQchatdumpingground-Datadog) with links explaining the details and an alternative approach.

Reply:
I see that Simon already pointed this out in his (now hidden) comment above.